install-bluetooth.sh

```diff
@@ -44,7 +44,7 @@ KillSignal=SIGUSR1
 [Install]
 WantedBy=multi-user.target
 EOF
-systemctl enable bt-agent.service
+systemctl enable bt-agent@hci0.service
```

Comment:
Does this work for other devices besides `hci0`?

Reply:
No it wouldn't. But my whole impression of `rpi-audio-receiver` is, that it is (and wants to be) rather opinionated about the setup: A Raspberry with Raspbian installed and 1 internal or external (USB) bluetooth device (which is referenced as `hci0`).

To actually consider different setups would mean to make the `install-bluetooth.sh` more complex. I do not have the feeling that this is intended, but it's @nicokaiser's call, I'm only describing my impression here.

Cheers
Thomas

Reply:
Like @TwizzyDizzy said: this project is rather opinionated, in terms of simplicity: I cannot support complex and custom setups, so the project assumes one Bluetooth device available (`hci0`). However, due to this the scripts are rather simple, so anyone can adjust the scripts for their needs. This is in my opinion way more practical than ambitious installer scripts (that change after each Raspbian update anyway).